modules/acquia_cms_common/src/RedirectHandler.php

```diff
@@ -89,7 +89,7 @@ public static function create(ContainerInterface $container) {
    * @param \Drupal\Core\Form\FormStateInterface $form_state
    *   The current form state.
    */
-  public static function submitForm(array &$form, FormStateInterface $form_state) : void {
+  public static function submitForm(array &$form, FormStateInterface $form_state): void {
     \Drupal::classResolver(static::class)->handleRedirect($form_state);
   }
```

Comment:
@apathak18 - I don't think we need this separate function, can we move all code from function `handleRedirect` to `submitForm`

Reply:
@chandan-singh7929 submitForm is static method called from acquia_cms_common_form_user_login_form_alter without discussion we cannot remove handleRedirect method.